src/components/feedback-message/FeedbackMessage.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { WarningIcon, InfoIcon } from '../icon';
+import { Tooltip, TooltipTrigger } from '../tooltip';
+import styles from './FeedbackMessage.css';
+
+const FeedbackMessage = ({ children, type, textColor, iconPosition, variant, tooltipChildren, className }) => {
+    const finalClassNames = classNames(
+        styles.feedbackMessage,
+        styles[type],
+        styles[iconPosition],
+        styles[variant],
+        className
+    );
+
+    const renderTooltip = () => {
+        if (!type) {
+            return;
+        }
+
+        return tooltipChildren ?
+            <TooltipTrigger tooltip={ <Tooltip placement="bottom" style={ { maxWidth: 286 } }>{ tooltipChildren }</Tooltip> }>
+                { <div className={ styles.iconWrapper }>{ renderIcon() }</div> }
+            </TooltipTrigger> :
+            renderIcon();
+    };
+
+    const renderIcon = () => type === 'error' ?
+        <WarningIcon className={ styles.icon } /> :
+        <InfoIcon className={ styles.icon } />;
+
+    return (
+        <div style={ { color: textColor } } className={ finalClassNames }>
+            { renderTooltip() }
+            { children }
+        </div>
+    );
+};
+
+FeedbackMessage.propTypes = {
+    children: PropTypes.string.isRequired,
+    variant: PropTypes.oneOf(['small', 'large']),
+    type: PropTypes.oneOf(['error', 'info']),
+    textColor: PropTypes.string,
+    iconPosition: PropTypes.oneOf(['left', 'right']),
+    tooltipChildren: PropTypes.node,
```

Comment:
I don't like this property name. I would name it just a `tooltip` and it could be a `node` or a `element`. If it's a element, it means I want to have full control of the tooltip, like so `tooltip={ <Tooltip placement="top">uhuhu</Tooltip>` }.

Reply:
I see your point. But why do we need this kind of flexibility at this point?

Reply:
I think we shouldn't require the developer to know that he needs to wrap the tooltip children in a tooltip component if he simply wants to, for example, pass a react element as the tooltip children. I feel like it's losing ease of use for flexibility that is not yet required.

At most I would say to pass a tooltip options object to provide the possibility of customization.